### PR TITLE
[Non-touch] Add a menu item to toggle a bookmark on the current page

### LINKED ISSUE #4260: Bookmark on kindle no touch
* KOReader version: v2018.10.07-beta
* Device: Kindle 4

How to add a new bookmark on kindle with keyboard like kindle 4?

### PR DESCRIPTION
**Pull request by onde2rock**
close  #4260

### CODE REVIEW

File: frontend/apps/reader/modules/readerbookmark.lua [hunk 1]
```diff
@@ -55,6 +55,14 @@ function ReaderBookmark:addToMainMenu(menu_items)
             self:onShowBookmark()
         end,
     }
+    if not Device:isTouchDevice() then
+        menu_items.add_bookmarks = {
+            text = _("Toggle bookmarks"),
```
Comment (poire-z) [inline frontend/apps/reader/modules/readerbookmark.lua]: I guess it should be singular `Toggle bookmark` (or `Toggle page bookmark` ?)
And for consistency, name the menu item index similar to its text, eg: `menu_items.toggle_bookmark` 

You could even add a function (untested):
```lua
function ReaderBookmark:isCurrentPageBookmarked()
    local pn_or_xp
    if self.ui.document.info.has_pages then
        pn_or_xp = self.view.state.page
    else
        pn_or_xp = self.ui.document:getXPointer()
    end
    return self:getDogearBookmarkIndex(pn_or_xp) and true or false
end
```
and use it as `text_func = function() return self:isCurrentPageBookmarked() and "Remove bookmark for current page" or "Bookmark current page"`

Comment (onde2rock) [reply frontend/apps/reader/modules/readerbookmark.lua]: Yes, that's better.

File: frontend/apps/reader/modules/readerbookmark.lua [hunk 2]
```diff
@@ -55,6 +55,14 @@ function ReaderBookmark:addToMainMenu(menu_items)
             self:onShowBookmark()
         end,
     }
+    if not Device:isTouchDevice() then
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readerbookmark.lua]: Should we hide this on touch devices? Pinging some touch users to see what they think:

@KenMaltby @ersidnd @cramoisi

Comment (poire-z) [reply frontend/apps/reader/modules/readerbookmark.lua]: Merging this (we can still show it later if there's feedback towards that).

**Pull request merged**